editor/blocks/quote/index.js

```diff
@@ -0,0 +1,44 @@
+const Editable = wp.blocks.Editable;
+const { query, html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: html( 'cite,footer' )
+	},
+
+	edit( attributes, onChange ) {
+		const { value, citation } = attributes;
+
+		// Concat strings parsed by hpq from paragraph values into one string.
+		const combinedValue = value.join( '' );
+		return (
+			<blockquote>
+				<Editable
+					value={ combinedValue }
+					onChange={ ( newValue ) => onChange( { newValue } ) } />
+				<cite>
+					<Editable
+						value={ citation }
+						onChange={ ( newValue ) => onChange( { newValue } ) } />
```

Comment:
the `onChange` is a callback we need to call to update the block attributes parsed originally by the `attributes` object above. This second `Editable` is responsible of editing the `citation` block attribute, so I think we should call `onChange( { citation: newValue } )`